recipes/pyretis/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pyretis" %}
+{% set version = "0.9.2.dev2" %}
```

Comment:
Is there a non-`dev` release that we can use (i.e. stable release).

Reply:
Hi,

So finally we have the 1.0.0 not dev version. How to make an upgrade? I tried to fork and update, it has then been merged, but I think only to my branch?

Reply:
You should send a pull request against https://github.com/conda-forge/pyretis-feedstock.